netbox_onboarding/netdev_keeper.py

```diff
@@ -32,6 +32,9 @@
 from .constants import NETMIKO_TO_NAPALM_STATIC
 from .exceptions import OnboardException
 
+logger = logging.getLogger("rq.worker")
+logger.setLevel(logging.DEBUG)
```

Comment:
Please can you remove this line, the level should come from the config

Reply:
Will remove!

Reply:
Removed.